guide/14-deep-learning/object-detection.ipynb

```diff
@@ -69,7 +69,7 @@
        "                        </a>\n",
```

Comment:
<ol><li>Spellcheck</li></ol><ul><li><em>given the input imagery, </em><strong><em>along with labelled</em></strong><em> vector data or classified images.&nbsp;</em></li><li><em>learning training samples are small </em><strong><em>sub images</em></strong><em>, called image chips, and contain the feature</em></li><li>package as well as by third-party deep learning <strong>libraries</strong>, such as TensorFlow</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>